src/api/index.ts

```diff
@@ -448,8 +448,8 @@ export function useApi(isOnlineProp: boolean = true): BackendApi {
 	>> => {
 
 		try {
-			const getPrivateDataResponse = await get('/user/session/private-data');
-			if (privateDataEtag && getPrivateDataResponse.headers['x-private-data-etag'] && getPrivateDataResponse.headers['x-private-data-etag'] === privateDataEtag) {
+			const getPrivateDataEtagResponse = await get('/user/session/private-data/etag');
```

Comment:
```suggestion
			const getPrivateDataEtagResponse = await get('/user/session/private-data?with_private_data=false');
```

So, you last suggestion would look like this, right?